src/components/Icons/Eye.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
```

Comment:
i recommend putting icons in /assets/icons wdyt?

Reply:
I think it could be moved but since it is a component and how I am working on it, it should be in its respective Components folder.